README.md

```diff
@@ -18,6 +20,24 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
 5.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.
 
+### Docker
+
+1. Download and install [Docker](https://docs.docker.com/engine/installation/)
```

Comment:
Docker Compose is shipped beside docker on osx and windows. I'll add instructions to install it also for Linux.